src/test/tests/analyzerUtils.test.ts

```diff
@@ -34,14 +35,12 @@ describe('Analyzer Utils Tests', async () => {
     });
 
     [
-        {
-            filePath: '',
-            expected: ''
-        }
+        path.join('file'),
+        path.join('somewhere','folder','file')
     ].forEach(testCase => {
-        it('Parse location file path test - ' + testCase.filePath, () => {
-            let result: string = AnalyzerUtils.parseLocationFilePath(testCase.filePath);
-            assert.deepEqual(result, testCase.expected);
+        it('Parse location file path test - ' + testCase, () => {
+            let result: string = AnalyzerUtils.parseLocationFilePath(`file://${testCase.replace(/['\\']/g, '/')}`);
+            assert.deepEqual(result, testCase);
```

Comment:
analyzerManager will response only in specific format not related on the os that it will be run on, we need to make sure to prepare the right input for the test case in window os